setup.py

```diff
@@ -16,6 +16,7 @@
         "django-celery==3.2.2",
         "django-environ==0.4.5",
         "django-extensions==2.1.0",
+        "django-prometheus>=1.0.15,<1.1",
```

Comment:
Why `<1.1`? Assuming django-prometheus follows semver, there shouldn't be any breakage in a minor version change.

Reply:
Oh, I see all the other deps also have very specific version requirements. In general, it's much better to define the allowed versions as broadly as possible and use per-deploy `requirements.txt` (or something equivalent) to get repeatability in production.

Reply:
Have updated the version. I think we should sort out the rest of the versioning in a separate PR.